book/src/dev/rfcs/0005-state-updates.md

```diff
@@ -600,23 +600,61 @@ order on byte strings is the numeric ordering).
 
 We use the following rocksdb column families:
 
-| Tree                 |                  Keys |                              Values |
-|----------------------|-----------------------|-------------------------------------|
-| `hash_by_height`     | `BE32(height)`        | `block::Hash`                       |
-| `height_by_hash`     | `block::Hash`         | `BE32(height)`                      |
-| `block_by_height`    | `BE32(height)`        | `Block`                             |
-| `tx_by_hash`         | `transaction::Hash`   | `(BE32(height) \|\| BE32(tx_index))`|
-| `utxo_by_outpoint`   | `OutPoint`            | `TransparentOutput`                 |
-| `sprout_nullifiers`  | `sprout::Nullifier`   | `()`                                |
-| `sapling_nullifiers` | `sapling::Nullifier`  | `()`                                |
-| `orchard_nullifiers` | `orchard::Nullifier`  | `()`                                |
-| `sprout_anchors`     | `sprout::tree::Root`  | `()`                                |
-| `sapling_anchors`    | `sapling::tree::Root` | `()`                                |
+| Column Family         | Keys                  | Values                               | Updates |
+|-----------------------|-----------------------|--------------------------------------|---------|
+| `hash_by_height`      | `BE32(height)`        | `block::Hash`                        | Never   |
+| `height_by_hash`      | `block::Hash`         | `BE32(height)`                       | Never   |
+| `block_by_height`     | `BE32(height)`        | `Block`                              | Never   |
+| `tx_by_hash`          | `transaction::Hash`   | `(BE32(height) \|\| BE32(tx_index))` | Never   |
+| `utxo_by_outpoint`    | `OutPoint`            | `transparent::Output`                | Delete  |
+| `sprout_nullifiers`   | `sprout::Nullifier`   | `()`                                 | Never   |
+| `sapling_nullifiers`  | `sapling::Nullifier`  | `()`                                 | Never   |
+| `orchard_nullifiers`  | `orchard::Nullifier`  | `()`                                 | Never   |
+| `sprout_anchors`      | `sprout::tree::Root`  | `()`                                 | Never   |
+| `sprout_incremental`  | `BE32(height)`        | `sprout::tree::NoteCommitmentTree`   | Delete  |
+| `sapling_anchors`     | `sapling::tree::Root` | `()`                                 | Never   |
+| `sapling_incremental` | `BE32(height)`        | `sapling::tree::NoteCommitmentTree`  | Delete  |
+| `orchard_anchors`     | `orchard::tree::Root` | `()`                                 | Never   |
+| `orchard_incremental` | `BE32(height)`        | `orchard::tree::NoteCommitmentTree`  | Delete  |
+| `history_incremental` | `BE32(height)`        | `zcash_history::NodeData`            | Delete  |
```

Comment:
Currently it needs `zcash_history::Entry`, though that can be computed from `NodeData` with a small computation. If converting it's the better approach, feel free to dismiss this.

Reply:
```suggestion
| `history_incremental` | `BE32(height)`        | `zcash_history::Entry`         | Delete  |
```